sys-apps/xinetd/xinetd-2.3.15.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A secure replacement for inetd"
+DESCRIPTION="powerful replacement for inetd"
+HOMEPAGE="https://github.com/xinetd-org/xinetd"
+COPYRIGHT="1992 by Panagiotis Tsirigotis
+	1998-2001 by Rob Braun"
+LICENSE="Xinetd"
+REVISION="1"
+SOURCE_URI="https://github.com/xinetd-org/xinetd/archive/xinetd-${portVersion//\./-}.tar.gz"
+CHECKSUM_SHA256="620b25f4ab4d72fdf32b13797156ea40df2049f1c07e640177e5fec544e9a94c"
+SOURCE_DIR="xinetd-xinetd-${portVersion//\./-}"
+PATCHES="xinetd-$portVersion.patch"
+
+ARCHITECTURES="?x86_gcc2 ?x86_64"
+
+PROVIDES="
+	xinetd = $portVersion
+	cmd:itox
+	cmd:xconv.pl
+	cmd:xinetd
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc
+	cmd:libtoolize
+	cmd:make
+	"
+
+BUILD()
+{
+	# if we don't have rpc/rpc.h then why does it still try to use it??
+	# let's change the #ifndef NO_RPC to #ifdef HAVE_RPC_RPC
+	sed -i 's/ndef NO_RPC/def HAVE_RPC_RPC/g' xinetd/*.c
```

Comment:
seds should go into PATCH()

Reply:
This is a leftover from the old bep file, but I can look into it to revert it to PATCH()

Reply:
reverted into PATCH()